pkg/hope/node_management.go

```diff
@@ -156,7 +157,8 @@ func CreateClusterMaster(log *logrus.Entry, node *Node, podNetworkCidr string, l
 		return err
 	}
 
-	joinCommand, err := ssh.GetSSH(otherMaster.ConnectionString(), "sudo", "kubeadm", "token", "create", "--print-join-command")
+	existingMasters := (*masters)[1:]
+	joinCommand, err := KubeadmGetClusterJoinCommandFromAnyMaster(&existingMasters)
```

Comment:
Uses the new function, which lets this have some more resilience, asking each other master to get the join token, rather than only asking the first.